src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable,AreaCalculator {
```

Comment:
```suggestion
public abstract class Figure implements Drawable, AreaCalculator {
```
use Ctrl Alt L to format your files before committing